Gemfile

```diff
@@ -5,7 +5,6 @@ gem "rake"
 gem "mocha"
 gem "minitest"
 gem "minitest-rg"
```

Comment:
If you move this into gemspec as .add_development_dependency all the gemfiles get dry :)